soul-spring-boot-starter/soul-spring-boot-starter-client/soul-spring-boot-starter-client-springcloud/src/test/java/org/dromara/soul/springboot/starter/client/springcloud/SoulSpringCloudClientConfigurationTest.java

```diff
@@ -53,10 +64,16 @@
     @Autowired
     private SoulSpringCloudConfig soulSpringCloudConfig;
 
+    @BeforeClass
+    public static void before() throws Exception {
+        PowerMockito.mockStatic(RegisterUtils.class);
+        PowerMockito.doNothing().when(RegisterUtils.class, "doRegister", Mockito.any(), Mockito.any(), Mockito.any());
+    }
```

Comment:
Do we have to import `powermock`? `Mockito ` also supports mock static methods. Here's a example [https://github.com/dromara/soul/blob/master/soul-client/soul-client-common/src/test/java/org/dromara/soul/client/common/utils/RegisterUtilsTest.java](https://github.com/dromara/soul/blob/master/soul-client/soul-client-common/src/test/java/org/dromara/soul/client/common/utils/RegisterUtilsTest.java) Can you try it?

Reply:
Modify the code by using Mockito

```java
MockedStatic<RegisterUtils> registerUtilsMockedStatic = mockStatic(RegisterUtils.class);
registerUtilsMockedStatic.when(() -> RegisterUtils.doRegister(any(), any(), any())).thenAnswer((Answer<Void>) invocation -> null);
```

Reply:
![image](https://user-images.githubusercontent.com/22816271/106379622-30f72d80-63e8-11eb-91bc-4fde39fe903a.png)